README.md

```diff
@@ -64,7 +64,17 @@ Landrush assigns your vm's hostname from either the vagrant config (see the `exa
 
 Every time a VM is started, its IP address is automatically detected and a DNS record is created that maps the hostname to its IP.
 
-If for any reason the auto-detection detects no IP address or the wrong IP address, or you want to override it, you can do like so:
+If for any reason the auto-detection detects no IP address or the wrong IP address you can control how it looks for it.
+
+You can start by excluding interfaces matching an array of regex patterns (the value shown here is the default):
+
+    config.landrush.exclude = [/lo[0-9]*/, /docker[0-9]+/, /tun[0-9]+/]
+
+Or, if you know which interface you need the IP of you can specify that too (default is none):
+
+    config.landrush.interface = 'eth0'
```

Comment:
> Deeper namespacing serves little purpose beyond aesthetics and it only makes it more prone to typos IMO.

Interesting. I think a better namespace would help, but for now let's just try to be consistent with existing names. I think I will then create a follow up version to potentially deprecate these _host_xyz_ settings for something less confusing (w/ or w/o additional depth in the names, that can be discussed then). But let's take this later.